editor.planx.uk/src/@planx/components/MultipleFileUpload/Modal.tsx

```diff
@@ -60,3 +72,78 @@ export const FileTaggingModal = (props: FileTaggingModalProps) => {
     </Dialog>
   );
 };
+
+interface SelectMultipleProps extends SelectProps {
+  name: string;
+  fileList: FileList;
+}
+
+const SelectMultiple = (props: SelectMultipleProps) => {
+  const { name, fileList } = props;
+  const [tags, setTags] = React.useState<string[]>([]);
+
+  const handleChange = (event: SelectChangeEvent<typeof tags>) => {
+    const {
+      target: { value },
+    } = event;
+    setTags(
+      // On autofill we get a stringified value.
+      typeof value === "string" ? value.split(",") : value
+    );
+  };
+
+  return (
+    <FormControl key={name} sx={{ display: "flex", flexDirection: "column" }}>
+      <InputLabel id="select-mutliple-file-tags-label">
+        What does this file show?
+      </InputLabel>
+      <Select
+        id="select-multiple-file-tags"
+        labelId="select-multiple-file-tags-label"
+        variant="standard"
+        multiple
+        value={tags}
+        onChange={handleChange}
+        IconComponent={ArrowIcon}
+        input={<Input />}
+        inputProps={{ name }}
+        renderValue={(selected) => selected.join(", ")}
```

Comment:
I've not actually looked at the docs for this component in a while, but I think this is the prop that will allow us to display neat little chips here 💪

Reply:
Yes exactly ! There's a nice example of just this in the docs here :nerd_face: https://mui.com/material-ui/react-select/#chip

Reply:
🎉 👏